database/migration/migrateData.ts

```diff
@@ -581,7 +581,7 @@ const migrateData = async () => {
 
     await DB.changeSchema(`
     ALTER TABLE data_table
-    ADD COLUMN data_view_code varchar;
+    ADD COLUMN IF NOT EXISTS data_view_code varchar;
```

Comment:
Nice! 

Just for other errors on - not related - other changes in schema. Could you also fix those 2 please?
After Rename TYPE is_reviewable to reviewability and update field in TABLE template_element

- Problem altering schema: type "public.is_reviewable_status" does not exist 

- Problem altering schema: column "is_reviewable" does not exist

Reply:
Yeah, I've actually tried to fix this a few times, but neither of those statements ("ALTER TYPE" and "RENAME COLUMN") have an "IF EXISTS / IF NOT EXISTS" option, so it's a pain. Apparently there's ways if you wrap it in a function and catch the error, or do an IF based on a schema lookup, but it's all a bit of a hassle for something tiny, so I've just added an option to silence the errors on "changeSchema" statements: d1f58396

Hope that works okay.

Reply:
Amazing, thanks!